draft-lenders-core-dnr.md

```diff
@@ -186,6 +188,61 @@ authentication server URI.
 Defining these SvcParamKeys, including their value formats and spaces, as well as the behavior
 definition for authenticator-domain-name field, shall be part of future work.
 
+## Scenarios
+
+Two example scenarios illustrate possible ways for which a solution should work;
+they are phrased in dialogue form rather than in terms of protocol elements to avoid bias on solutions.
+
+### Neighbor discovered server with EDHOC credential
+
+In which the DoC server restricts access by network address (or not at all),
+and the client trusts its router to advertise a suitable Encrypted DNS server.
+
+1. Client: Joins a network.
+2. Local router: Sends out an IPv6 Router Advertisement (RA) with Encrypted DNS option (see {{Section 6 of -dnr}}).
+
+   Next to the network address, this conveys Service Parameters indicating DoC
+   as well as a CWT Claims Set (CCS, {{rfc8392}}) that is to be used as an EDHOC credential.
```

Comment:
This reference is not defined. I would prefer to add it to the informative references in the metadata, but `{{?RFC8392}}` would also be ok.